README.md

```diff
@@ -26,7 +26,24 @@ and provides step-by-step migration instructions to v3.
 To test your universal JavaScript code in Node.js, browser and react-native environments,
 visit our [code samples repo](https://github.com/aws-samples/aws-sdk-js-tests).
 
-## Getting started
+# Table of Contents
+
+1. [Getting Started](#getting-started)
+1. [New Features](#new-features)
+   1. [Modularized packages](#modularized-packages)
+   1. [API consistency changes](#api-changes)
+      1. [Configuration](#configuration)
+      1. [Middleware Stack](#Middleware)
+   1. [How to upgrade](#other-changes)
+1. [Install from Source](#install-from-Source)
+1. [Giving feedback and contributing](#giving-feedback-and-contributing)
+1. [High Level Concepts in V3](#high-level-concepts)
+   1. [Generated Packages](#generated-code)
+   1. [Paginators](#paginators)
+   1. [Abort Controller](#abort-controller)
+   1. [Middleware Stack](#middleware-stack)
```

Comment:
The high level concepts should appear after New Features, and before Install from source.
```suggestion
1. [High Level Concepts in V3](#high-level-concepts)
   1. [Generated Packages](#generated-code)
   1. [Paginators](#paginators)
   1. [Abort Controller](#abort-controller)
   1. [Middleware Stack](#middleware-stack)
1. [Install from Source](#install-from-Source)
1. [Giving feedback and contributing](#giving-feedback-and-contributing)
```

Reply:
Unresolving as "High level conepts" are yet to be moved above "Install from source"